.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,108 @@
+name: "Report a Linux Issue"
+description: "Create a report for a runtime related Linux Issue"
+labels: ["os:linux"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Linux Distribution and Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - Window Manager and version
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - Linux version:
+          - Source:
+          - Introduced in version:
+          - Window Manager and version:
+          - GPU model, driver and version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
+          Try to reproduce your issue with `--no-config` first. If it isn't reproducible
+          with `--no-config` try to find out which option or script causes your issue.
+
+
+          If the issue is render performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+
+          For decoding performance-related issues, additionally try to enable hardware
+          decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file with `--log-file=output.txt`, which is our preferred way, and attach it to
+          the issue.
+
+
+          If you have trouble producing a log file, you can alternatively use `-v -v` and save the
+          terminal output to a file.
+
+
+          In the case of a crash, please provide a backtrace.
```

Comment:
How, where? We have to assume user know nothing. This needs to be explained. We could link to https://wiki.archlinux.org/title/Debugging/Getting_traces

Reply:
do we have something similar for windows?

Reply:
For Windows they should attach `.dmp` file from `%LOCALAPPDATA%\CrashDumps` location. It is important to share exact binary `.exe` and `.pdb` file so we can read that. 

The most common builds for windows (https://github.com/shinchiro/mpv-winbuild-cmake/) have `.pdb` files and are compatible now. Best would be to use our internal CI MSVC build. Though it might be missing some dependencies currently.